src/db.js

```
@@ -0,0 +1,10 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
```

Sequelize constructor uses several environment variables. Consider validating these values (or providing sensible defaults) before constructing Sequelize so the app fails fast with a clear error when DB config is missing.